x-pack/plugins/triggers_actions_ui/public/application/sections/alert_summary_widget/components/alert_summary_widget_compact.stories.tsx

```diff
@@ -20,5 +21,9 @@ export const Compact = {
     chartProps: mockedChartProps,
     timeRangeTitle: 'Last 30 days',
     onClick: action('clicked'),
+    dependencyProps: {
+      baseTheme: LIGHT_THEME,
```

Comment:
This fixed the error following error but we still need to fix the style, I created an [issue](https://github.com/elastic/kibana/issues/183174) for it.

![image](https://github.com/elastic/kibana/assets/12370520/4368811e-0597-4820-964c-0a209d413944)